mercury/test_models.py

```diff
@@ -1,25 +1,32 @@
 from django.test import TestCase
-from mercury.models import SimulatedData
+from mercury.models import (
+    TemperatureSensor,
+    # AccelerationSensor,
+    # WheelSpeedSensor,
+    # SuspensionSensor,
+    # FuelLevelSensor,
+)
 import datetime
 
 TEST_TEMP = 999.0
 
 
-def create_simulated_data():
-    SimulatedData.objects.create(
+# TODO
+def create_simulated_temp():
+    TemperatureSensor.objects.create(
         temperature=TEST_TEMP, created_at=datetime.datetime.now()
     )
 
 
-class TestSimulatedData(TestCase):
+class TestTemperatureSensor(TestCase):
     def setUp(self):
-        create_simulated_data()
+        create_simulated_temp()
 
     def test_vehicle_temp(self):
-        foo = SimulatedData.objects.get(temperature=TEST_TEMP)
+        foo = TemperatureSensor.objects.get(temperature=TEST_TEMP)
         self.assertEqual(foo.temperature, TEST_TEMP)
 
-    def test_vehicle_temp_method(self):
-        foo = SimulatedData()
-        foo.temperature = 987
-        self.assertEqual(foo.temp(), 987)
+    # def test_vehicle_temp_method(self):
+    #     foo = TemperatureSensor()
+    #     foo.temperature = 987
+    #     self.assertEqual(foo.temp(), 987)
```

Comment:
If this test is no longer needed, we can remove it (no need to leave it here commented).